src/Symfony/Component/Debug/DebugClassLoader.php

```diff
@@ -229,11 +232,12 @@ private function checkClass($class, $file = null)
                 }
             }
 
-            // Inherit @final and @internal annotations for methods
+            // Inherit @final, @internal and @param annotations for methods
             self::$finalMethods[$name] = array();
             self::$internalMethods[$name] = array();
+            self::$annotatedParameters[$name] = array();
             foreach ($parentAndTraits as $use) {
```

Comment:
Do we want to support `@param` in interfaces (imo yes)? If so we should add a separate foreach to inherit `annotatedParameters`.

Reply:
yep we want to report `@param` added to interfaces for sure

Reply:
done